### PR TITLE
updated readme, also for Nick to test pull req

### PR DESCRIPTION
**Pull request by seunomonije**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Google Student Training in Engineering Program
 
+### Let's get this baby started..I'm at Google!!!
```
Comment (jpnickolas) [inline README.md]: nit: make this "... "
```suggestion
### Let's get this baby started... I'm at Google!!!
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Start</h1>
       <p>This is my portfolio.</p>
+      <p>My name is Seun aka computabeast.</p>
+      <img src = "images/IMG_9177.jpg" style = "height: 100px; width: auto;"/>
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/index.html]: Prefer using css over the style attribute. I suggest giving this image a relatively unique class name.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,7 +1,14 @@
+html {
+  background: url(images/wtfbet.png) no-repeat center center fixed;
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/style.css:4]: lol

**Review by jpnickolas — CHANGES_REQUESTED** [review]
IMG_9177.jpg is a pretty vague name. I suggest changing it to something clearer.

**Comment by seunomonije** [comment]
Screenshot 2020-05-26 at 11 33 42 AM

Ss of the page...The content is off because I worked on it and commented the irrelevant parts out in the time I submitted this req to now

**Review by jpnickolas — APPROVED** [review]
Thanks!

**Review by jessicaslaughter — APPROVED** [review]
nice!

**Pull request merged**